functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
```

Comment:
Great work! Think about the time complexity of your solution. Right now its O(n2). Here a good implementation. 
'''python
for a in range:
'''

Reply:
### **Nope**